### ISSUE TITLE
explore cameras

### ISSUE THREAD

**Issue by tabiosg** [umrover/embedded-testbench#163]
yes

**Comment by sashreek1** [comment]
Here's the new ROS camera code (untested):
https://github.com/tabiosg/mrover-ros/tree/cameras_ros

Camera code before switching to ROS:
https://github.com/umrover/mrover-workspace/tree/main/jetson/cameras

**Comment by tabiosg** [comment]
@sashreek1 wait are u telling them to create a program using jetson-utils?

**Comment by tabiosg** [comment]
imo it would be useful if they listed all the options for camera programs instead

**Comment by sashreek1** [comment]
Yeah agreed. Try looking into gstreamer or OpenCV for this. These are just options try looking at other options too.

**Comment by qhdwight** [comment]
jetson-utils uses gstreamer. What is wrong with it? It will actually fit perfectly into our code base since it uses CMake

**Comment by qhdwight** [comment]
https://github.com/dusty-nv/jetson-utils/blob/35593c5ad2c1c62f6f0166d6581945fb58fd1f7b/camera/v4l2Camera.cpp#L86

There is a timeout parameter. This should figure the freezing issue.

**Comment by tabiosg** [comment]
@sashreek1 Ok I talked with Quintin and we should stick with jetson-utils for now. We can look at other alternatives too but for now just write this program
